etsin_finder/frontend/js/components/qvain/files/externalFiles.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import Translate from 'react-translate-component'
+import {
+  FilePickerButtonInverse,
+  FilePickerButtonText,
+  LinkIcon,
+  ChevronIcon
+} from '../general/buttons'
+
+class ExternalFiles extends Component {
```

Comment:
Stub of the ```ExternalFiles``` component that will be properly implemented later (as CSCETSIN-416).